JeongGod/7week/programmers/12973.py

```diff
@@ -0,0 +1,15 @@
+def solution(s):
+    answer = -1
+    st = []
+    
+    for i in s:
+        if not st:
+            st.append(i)
+            continue
+        if i == st[-1]:
+            st.pop()
```

Comment:
```suggestion
        if st and i == st[-1]:
            st.pop()
```
조건문 앞에부터 걸리는거 활용해서 더 짧게는 이렇게도 가능하겠네용